tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -28,6 +30,26 @@ import (
 	dep "istio.io/istio/tools/istio-iptables/pkg/dependencies"
 )
 
+type Runner struct {
```

Comment:
I'm not very happy about this name.
Can we find a more specific name? How about `IptablesConfigurator`? 🤔

Reply:
Changed to `IptablesConfigurator`